src/KlaviyoAPI.php

```diff
@@ -495,7 +495,7 @@ protected function getSpecificCurlOptions($options)
      * @param array $headers
      * @return array
      */
-    protected function formatCurlHeaders(array $headers)
+    protected function formatCurlHeaders($headers)
```

Comment:
Nit: I see the other methods use spacing before and after, why not be consistent
```suggestion
    protected function formatCurlHeaders( $headers )
```